modules/encyclopedia.nf

```diff
@@ -80,14 +100,9 @@ process ENCYCLOPEDIA_GLOBAL {
 
     script:
     """
-    source ~/.bashrc
-    mkdir logs
-    gunzip ${local_feature_files}
+    gzip -df ${local_feature_files}
```

Comment:
Why are these feature files being zipped here? Are they being used? If not, can we zip them somewhere else and not pass the feature files into this process?

Reply:
Yes, these are being used by EncyclopeDIA.

Here, `gzip -d` is equivalent to `gunzip`, but yielded more informative error messages when I was debugging for some reason. The `-f` forces it to follow the symlinks that NextFlow uses locally.